portal/static/js/src/modules/TnthAjax.js

```diff
@@ -1133,7 +1134,7 @@ export default { /*global $ */
             return false;
         }
         params = params || {};
-        this.sendRequest("/api/user/" + userId + "/questionnaire_bank", "GET", userId, {data: {as_of_date: completionDate}, sync: params.sync}, function(data) {
+        this.sendRequest("/api/user/" + userId + "/questionnaire_bank", "GET", userId, {data: {as_of_date: completionDate}, sync: params.sync, max_attempts: MAX_ATTEMPTS}, function(data) {
```

Comment:
@achen2401 is this change necessary?  Wouldn't we pick up the correct `default` without including, as defined on line #21?

Reply:
@pbugni You are right, I was just thinking that. I think it is safe to remove the change here.